util/spellcheck.sh

```diff
@@ -8,7 +8,7 @@ fi
 ROOT=$(git rev-parse --show-toplevel)
 
 # This gets us only english .md files at the moment, but is clearly brittle
-MARKDOWN_FILES=$(find "$ROOT" -name '*.md' -not -path '**/ptBR/**' -not -path '**/*-ptBR.md' -not -path '**/ja/**')
+MARKDOWN_FILES=$(find "$ROOT" -name '*.md' -not -path '**/ptBR/**' -not -path '**/*-ptBR.md' -not -path '**/ja/**' -not -path '**/.*/**')
```

Comment:
The lang paths could be removed here now that we've retired those paths.

Reply:
There was still one non-english doc hidden in `hardware`, that's why I didn't touch that part.
It's removed now. The "hardware" part of KMK isn't maintained anyway (and imo shouldn't be part of the firmware repository).